docs/locale/fr/LC_MESSAGES/userguide/new_organization.po

```diff
@@ -6,162 +6,285 @@ msgid ""
-"that organization using an organization bootstrap URL. You can read about "
-"this in the :ref:`the Hosting server <doc_adminguide_hosting>` part."
+#: ../../userguide/new_organization.rst:38
+msgid "Select ``Create my organization on Parsec``."
+msgstr "Sélectionnez ``Créer mon organisation sur Parsec``."
+
+#: ../../userguide/new_organization.rst:39
+msgid "Log in with your customer account."
+msgstr "Connectez-vous avec votre compte client."
+
+#: ../../userguide/new_organization.rst:40
+msgid "Enter the organization name (this name cannot be changed later)."
 msgstr ""
-"Une autre possibilité (bien que plus technique) est d'héberger votre propre "
-"serveur Parsec et de créer l'organisation sur celui-ci via la CLI Parsec. "
-"Vous pouvez rejoindre cette organisation en fournissant cet URL de bootstrap "
-"à Parsec. Vous pouvez lire plus à ce sujet dans la partie :ref:`Héberger le "
-"serveur <doc_adminguide_hosting>`."
+"Saisissez le nom de l'organisation (ce nom ne pourra pas être modifié "
+"ultérieurement)."
 
-#: ../../userguide/new_organization.rst:29
+#: ../../userguide/new_organization.rst:41
 msgid ""
-"Once you start the process, you are asked for an Email and an Organization "
-"Name. This email address will be used to contact you later."
+"Choose the preferred authentication method (this can be changed later from "
+"your profile)."
 msgstr ""
-"Une fois que vous avez démarré le processus, il vous sera demandé un Email "
-"et un Nom d'Organisation. C'est cette adresse email qui sera utilisée pour "
-"vous contacter."
+"Choisissez la méthode d'authentification préférée (vous pouvez la modifier "
+"ultérieurement à partir de votre profil)."
 
-#: ../../userguide/new_organization.rst:-1
-msgid "Organization create process"
-msgstr "Processus de création d'organisation"
+#: ../../userguide/new_organization.rst:43
+msgid "Check the information entered and confirm."
+msgstr "Vérifiez les informations saisies et confirmez."
 
-#: ../../userguide/new_organization.rst:35
-msgid ""
-"Creating an organization through the Parsec client register yourself as its "
-"first user and device (this step is called the bootstrap). This is why you "
-"must enter your Username, Device name, and your Password."
-msgstr ""
-"Créer une organisation depuis le client Parsec vous enregistre comme son "
-"premier utilisateur et appareil (cette étape est appelée le bootstrap). "
-"C'est pourquoi vous devez entrer votre nom d'utilisateur, d'appareil, et mot "
-"de passe."
+#: ../../userguide/new_organization.rst:45
+msgid "🚀 Congratulations, you've just created your organization on Parsec!"
+msgstr "🚀 Félicitations, vous venez de créer votre organisation sur Parsec !"
 
-#: ../../userguide/new_organization.rst:-1
-msgid "Organization bootstrap process"
-msgstr "Processus de démarrage d'une organisation"
+#: ../../userguide/new_organization.rst:47
+#: ../../userguide/new_organization.rst:66
+#: ../../userguide/new_organization.rst:78
+msgid "|first_steps_note|"
```

Comment:
![2024-09-19_screenshot_new_org](https://github.com/user-attachments/assets/92cc9980-1119-45f8-a4bc-09ca3bf2e276)
I'm not sure where to put that, but the first step note appear to not be translated

Reply:
Good catch! It seems to be an issue with the `replace` directive, I'll check that.

Reply:
[`8af7270`](https://github.com/Scille/parsec-cloud/pull/8387/commits/8af72704163b00dc0eb1006ad9ed89e521f1731f)